vuepress/docs/next/docs/create/component-gen-overview.md

```diff
@@ -1,42 +1,44 @@
 # Entando Component Generator
 
 
-The Entando Component Generator, powered by JHipster, is a tool that gives developers the ability to quickly generate Entando components to extend an Entando Application.
+The Entando Component Generator, powered by JHipster, is a tool that quickly generates Entando components to extend an Entando Application.
 
-The component generator takes advantage of the concept of [JHipster blueprints](https://www.jhipster.tech/modules/creating-a-blueprint/) to provide powerful templating and standard generation patterns.
+The Component Generator leverages the [JHipster blueprint](https://www.jhipster.tech/modules/creating-a-blueprint/) to provide powerful templating and standard generation patterns.
 
 ## Objectives
 
-The component generator has two main objectives
+The Component Generator has two main objectives, acceleration and standardization, which are examined below in greater detail.
 
-* Accelerate development
-  * Automate the creation of microservices and micro frontends with code that runs without changes. Start from running software.
-  * Code is meant to be developed on and changed by developers. Ready for customization.
-  * Using standard tools, technologies, and patterns that can be consumed by developers who are not familiar with the underlying blueprint
+### Accelerate Development
 
-* Standardize microservice creation and technology choices
-  * Encode business standards for networking, monitoring, build tools, and distribution into a blueprint to ensure that developers are starting from a technical point that matches your teams requirements
-  * Ensure code follows standards and practices of your teams including code formatting, testing tools, static analysis tools, package structures, and other rules specific to your organization
-  * Enable teams to quickly recognize and adopt code from system integrators and other third parties because the patterns and standards are encoded in the starting point of your applications
+- Begin with running software: By automating the creation of microservices and micro frontends, the user is able to bypass the manual development required to build the default configuration of an Entando project.
+- Customize as desired: The generated code is ready to be developed on and modified to enable features and capabilities.
+- Avoid a learning curve: The Entando Blueprint uses standard tools, technologies, and patterns that can be consumed by developers who are not familiar with the underlying technology.
+
+### Standardize Technology 
+
+- Guarantee technology compliance: The Component Generator encodes business standards for networking, monitoring, build tools, and distribution into the Entando Blueprint to ensure that business requirements are satisfied.
```

Comment:
I really like the format with the important point out front separated by a colon.  Just wondering if 'guarantee' might be a wee strong

Reply:
normally i would agree, but the outcome is a sure thing by definition, e.g. if component generator is certified as compliant to industry networking standards, the business requirement for that is satisfied. having a standards compliance background i am sure of this ;)